server/src/model.ts

```diff
@@ -2408,17 +2409,18 @@ export async function getFolderContent({
   loggedInUserId: number;
 }) {
   const notMe = ownerId !== loggedInUserId;
+  let parentFolder = null;
 
   if (folderId !== null) {
     // if ask for a folder, make sure it exists and is allowed to be seen
-    await prisma.content.findUniqueOrThrow({
+    parentFolder = await prisma.content.findUniqueOrThrow({
       where: {
         ownerId,
         id: folderId,
         isDeleted: false,
         isPublic: notMe ? true : undefined,
       },
-      select: { id: true },
+      select: { parentFolderId: true },
```

Comment:
You can save yourself the hassle in the front end by making the variable be `parentFolderId`, and the setting it as
```
parentFolderId = (await prisma..content.findUniqueOrThrow{( 
   ...
 }).parentFolderId